web/src/pages/HomePage/HomePage.jsx

```diff
@@ -1,14 +1,16 @@
-import {Link, Redirect, routes} from '@redwoodjs/router'
+import { Link, Redirect, routes } from '@redwoodjs/router'
 import { MetaTags } from '@redwoodjs/web'
+import { toast, Toaster } from '@redwoodjs/web/toast'
+
 import { useAuth } from 'src/auth'
 
 const HomePage = () => {
   const { currentUser, isAuthenticated } = useAuth()
-
+  toast.success(`Login successful, welcome ${currentUser.email}`)
```

Comment:
Here the toast is always played whenever a user visits that page. This means any time they refresh they will see the same toast. We only want it to happen right after they log in. Check this example for creating a dynamic toast, and the ability to add more readable error messages to oauth.js: https://redwoodjs.com/docs/how-to/oauth#better-error-handling